x-pack/plugins/endpoint/server/routes/endpoints.test.ts

```diff
@@ -0,0 +1,183 @@
+/*
```

Comment:
Would it be worth creating an integration test to make sure the ES query works against real data in ES? I believe these tests just mock ES out correct?

Reply:
> Would it be worth creating an integration test to make sure the ES query works against real data in ES? I believe these tests just mock ES out correct?

I will add the integration test, but it is not going to replace the Unit test. They are both needed.